src/Components/Photo_Of_The_Week_Archive_Controller.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace UCSC\Blocks\Components;
+
+use UCSC\Blocks\Components\Traits\With_Image_Size;
+use UCSC\Blocks\Object_Meta\Photo_Of_The_Week_Meta;
+use UCSC\Blocks\Post_Types\Photo_Of_The_Week\Photo_Of_The_Week;
+
+class Photo_Of_The_Week_Archive_Controller {
+	
+	use With_Image_Size;
+	
+	public function get_query( $paged ): \WP_Query {
+		return new \WP_Query( [
+			'post_type'      => Photo_Of_The_Week::NAME,
+			'post_status'    => 'publish',
+			'posts_per_page' => get_option( 'posts_per_page' ),
+			'orderby'        => 'date',
+			'order'          => 'DESC',
+			'paged'          => $paged,
+		] );
+	}
+    
+    public function get_image()
+    {
+        $image = get_field( Photo_Of_The_Week_Meta::IMAGE, get_the_ID() );
+        
+        if ( empty( $image ) || $image['ID'] < 1 ) {
+            return '';
+        }
+
+        $image_meta = wp_get_attachment_metadata( $image['ID'] );
+        
+        $image_data = array_merge( [
+            'id' => $image['ID'],
+            'url' => $image['url'],
+        ], $image_meta );
+        
+        return sprintf( '<img src="%s" srcset="%s" alt="%s" />', 
+            $image['url'], 
+            $this->build_srcset( $image_data ),
+            get_the_title( get_the_ID() )
+        );
+    }
+	
+	public function get_pagination( \WP_Query $query, int $paged ) {
```

Comment:
Yes we do. This is a component of the custom loop. We aren't in the theme and I can't wire custom loop with ACF meta to regular pagination block. 
The custom loop block is called [here](https://github.com/ucsc/ucsc-custom-functionality/pull/34/files#diff-39b550415f369728bb039d2b5157409db9a562f288805335df2bdb0e09c04644R8)

Reply:
@bjcooper just so you are aware, you will want to attempt to migrate the styles from the theme's pagination to this. this will require some implementation of classes as well as modification to the pagination stylesheet in the theme. The structures are slightly different.